### PR TITLE
[risk=no] Allele population chart

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
image

### CODE REVIEW

**Comment by SrushtiGangireddy** [comment]
image

File: public-ui/src/app/data-browser/views/genomic-view/components/variant-expanded.component.tsx [hunk 1]
```diff
@@ -1,34 +1,201 @@
...
+            variantPopulationDetails.push({'Ancestry': 'Total', 'AlleleCount': variantDetails.afrAlleleCount +
+            variantDetails.easAlleleCount + variantDetails.eurAlleleCount + variantDetails.amrAlleleCount +
+            variantDetails.midAlleleCount + variantDetails.sasAlleleCount + variantDetails.othAlleleCount,
+                'AlleleNumber': variantDetails.afrAlleleNumber + variantDetails.easAlleleNumber + variantDetails.eurAlleleNumber +
+                variantDetails.afrAlleleNumber + variantDetails.midAlleleNumber + variantDetails.sasAlleleNumber +
+                variantDetails.othAlleleNumber,
+                'AlleleFrequency': variantDetails.afrAlleleFrequency + variantDetails.easAlleleFrequency +
+                variantDetails.eurAlleleFrequency + variantDetails.afrAlleleFrequency + variantDetails.midAlleleFrequency +
+                 variantDetails.sasAlleleFrequency + variantDetails.othAlleleFrequency});
+        }
```
Comment (mhigbyflowers) [inline public-ui/src/app/data-browser/views/genomic-view/components/variant-expanded.component.tsx]: I think this shouldn't be in the render()

Comment (SrushtiGangireddy) [reply public-ui/src/app/data-browser/views/genomic-view/components/variant-expanded.component.tsx]: Moved this to constants service file.

File: public-ui/src/app/data-browser/views/genomic-view/components/variant-row.component.tsx [hunk 2]
```diff
@@ -44,53 +45,79 @@ const styles = reactStyles({
 
 
 interface Props {
-    varData: Variant;
+    variant: Variant;
+}
+
+interface State {
+    variantCliked: boolean;
```
Comment (dolbeew) [inline public-ui/src/app/data-browser/views/genomic-view/components/variant-row.component.tsx]: Nit: 
```suggestion
    variantClicked: boolean;
```

**Review by dolbeew — APPROVED** [review]

**Pull request merged**